### PR TITLE
Fix SPI comments + U20 pins cleanup

### PR DESCRIPTION
**Pull request by tpruvot**

### CODE REVIEW

File: Marlin/src/pins/stm32/pins_LONGER3D_LK.h [hunk 1]
```diff
@@ -16,28 +16,28 @@
...
+#define DISABLE_JTAG             //  We free the jtag pins (PA15) but keep STLINK
                                  //  Release PB4 (STEP_X_PIN) from JTAG NRST role.
-//#define DISABLE_DEBUG          //  > Hobi : We still want to debug with STLINK...
-#define DISABLE_JTAG             //  we free the jtag pins (PA15) but keep STLINK
-
 //
 // Limit Switches
 //
-#define X_STOP_PIN         PC1   // pin 16
+#define X_MIN_PIN          PC1   // pin 16
```
Comment (thinkyhead) [inline Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: If all these endstop pins have headers on the board then all these pins can be uncommented.

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: https://aliexpress.com/item/32944556459.html

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: yes, we have 6 endstop connectors

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: but... Zmax can be used for the bltouch servo in some cases

Comment (thinkyhead) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: I agree, the unmarked board makes it hard to tell. But the plugs on the left look like they are intended for 6 endstops.

Comment (thinkyhead) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: Aha, I see you already answered my question.

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: please let them commented please, they are not wired to sensors, and E is using XMAX

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: but they are named like that on the board itself...

Comment (tpruvot) [reply Marlin/src/pins/stm32/pins_LONGER3D_LK.h:40]: https://drive.google.com/file/d/1-q67OfhRc1eztqRmlJGDw0tPHCFmnhj7/view?usp=sharing

**Comment by tpruvot** [comment]
you left the typo also in spi comment...

**Pull request merged**